.rubocop.yml

```diff
@@ -0,0 +1,41 @@
+require: rubocop-rspec
+
+RSpec.Metrics/LineLength:
+      #Max: 120
+      Enabled: false
+
+RSpec/MessageSpies:
+  Enabled: false
+
+Metrics/BlockLength:
+  Max: 160
+
+Metrics/LineLength:
+  Max: 100
+
+
+
+Metrics/MethodLength:
+  Max: 15
+
+Metrics/AbcSize:
+  Max: 15
+
+Metrics/CyclomaticComplexity:
```

Comment:
rubocop default us 6, do not change this value
as well as other defaults, try to fix the code instead